packages/react-server-examples/bike-share/package.json

```diff
@@ -7,35 +7,21 @@
   "author": "Doug Wade <doug.wade@redfin.com>",
   "scripts": {
     "clean": "rm -rf build __clientTemp",
-    "start": "npm run clean && npm run styles && react-server start",
-    "styles": "node-sass styles/index.scss build/styles/index.css && node-sass styles/network.scss build/styles/network.css",
-    "test": "xo && nsp check"
+    "start": "npm run clean && react-server start --hot",
+    "test": "nsp check"
   },
   "license": "Apache-2.0",
   "dependencies": {
-    "babel-runtime": "^6.6.1",
+    "babel-runtime": "^6.11.6",
     "react": "~0.14.2",
     "react-dom": "~0.14.2",
     "react-server": "^0.4.5",
     "react-server-cli": "^0.4.5",
+    "sass-loader": "^4.0.0",
```

Comment:
Have you run `npm i` in `packages/react-server-cli` since the dep was added there?